doc/user_manual/existing_interfaces.tex

```diff
@@ -4741,4 +4741,113 @@ \subsubsection{Additional Information}
 searches for the default SPH solver parameter name: '\texttt{NIISphSolver\_1}'. If the SPH solver name is changed in the
 Neutrino input file, the Neutrino interface must also be changed. Similarly, the Neutrino interface searches for the
 output in the default Measurement field name: '\texttt{MeasurementField\_1}'. Again, this would need to modified in the interface
-if the measurement field name was changed.
\ No newline at end of file
+if the measurement field name was changed.
+
+\subsection{Prescient Interface}
+\label{subsec:prescientInterface}
+
+\subsubsection{General Information}
+The Prescient Interface is used to run the open source Prescient
+production cost modeling platform available from
+\url{https://github.com/grid-parity-exchange/Prescient}
+
+This allows inputs to be perturbed and data to be read out.
+
+\subsubsection{Sampler}
+
+For perturbing inputs, the sampled variable needs to be placed inside
+of \$( )\$ like \verb'$(var)$'. The sampled variable can have a
+constant added or a multiplication factor like \verb'$(var+3.2)$' or
+\verb'$(var*2.1)$' or \verb'$(var*5.0+7.0)$' or \verb'$(a*-2.0)$'
+These can be placed in any of the .dat or .csv files that are listed
+in the \xmlNode{Files} section as \xmlAttr{type="PrescientInput"}
```

Comment:
Josh, can you add an example of the Sampler block?

Reply:
I added an example of a sampler block.